class-instant-articles-post.php

```diff
@@ -602,6 +602,7 @@ public function to_instant_article() {
 
 		// Initialize transformer
 		$file_path = plugin_dir_path( __FILE__ ) . 'rules-configuration.json';
+		$file_path = apply_filters('instant_articles_transformer_rules_configuration_json_file_path', $file_path );
```

Comment:
great stuff.  many thanks for implementing this little code change/pull request.  I glad you like it.